WebApiExtended/CacheController.cs

```diff
@@ -58,16 +60,39 @@ public void Clear(string cacheName, string itemKey)
                 throw;
             }
         }
+
         /// <summary>
-        /// Gets names of all registered cache instances
+        /// Gets names and configuration of all registered cache instances
         /// </summary>
         [HttpGet]
         [Route("")]
-        public IEnumerable<string> GetRegisteredCacheNames()
+        public object GetRegisteredCacheNamesWithPolicies(bool includeConfig = false)
```

Comment:
@AwMalka  breaking change. return value type is changed. I think this is also true for the other web api

Reply:
it still returns IEnumerable<string>
just when new query item "includeConfig" is sent with value of TRUE
the output type will differ

we can separate/remove this functionality if needed
please discuss this with Zohar and let me know